src/mmw/js/src/modeling/models.js

```diff
@@ -751,6 +758,7 @@ var ScenariosCollection = Backbone.Collection.extend({
             aoi_census: aoi_census
         });
 
+        scenario.set('waitToRefresh', true);
```

Comment:
Maybe, but the problem is that that `parse` function is being called by the framework, so it is not clear (to me anyway) exactly when and where that is called.  I wanted to be as conservative as possible.